style.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Let's create a new directory inside the `public/css` and add a `global.css` file where we add this code.

Also, let's delete the `padding` too and set them to 0 for the `html` element as well.